chomp shebang line from python source files

cgoldberg opened the pull request:
removed shebang line from python source files.
see: #612

p0deje commented:
You should just remove https://github.com/SeleniumHQ/selenium/blob/master/Rakefile#L709 and re-run build task.

andreastt commented:
This is auto-generated from the build target in _Rakefile_ which should be update instead.

cgoldberg commented:
ok... I've updated Rakefile, and removed the auto-generation of the shebangs... please re-review.

besides removing shebangs from the build target in Rakefile, they should also be removed from the raw source files.
